GliaWidgetsTests/Mocks/UIViewControllerCoordinator.Mock.swift

```diff
@@ -0,0 +1,20 @@
+@testable import GliaWidgets
+import UIKit
+
+extension UIViewControllerCoordinator {
+    static let mock: UIViewControllerCoordinator = Mock()
+
+    private final class Mock: UIViewControllerCoordinator {
+        init() {
+            super.init(
+                environment: .init(
+                    uuid: { UUID() }
+                )
+            )
+        }
+
+        override func start() -> UIViewController {
+            UIViewController()
```

Comment:
do we need to call `super.start` here or would we like to test it without base implementation?

Reply:
Same as above. There was a reason that the coordinator needs to be a class and not a protocol, I forgot exactly what it was.